src/plugins/home/public/application/components/guided_onboarding/getting_started.tsx

```diff
@@ -0,0 +1,103 @@
+ */
+
+import React, { useEffect } from 'react';
+import {
+  EuiFlexGrid,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiLink,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+
+import { METRIC_TYPE } from '@kbn/analytics';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { i18n } from '@kbn/i18n';
+import { KibanaPageTemplate } from '@kbn/shared-ux-components';
+
+import { getServices } from '../../kibana_services';
+import { UseCaseCard } from './use_case_card';
+
+import './getting_started.scss';
+
+export const GettingStarted = () => {
+  const { application, trackUiMetric, chrome } = getServices();
+
+  useEffect(() => {
+    chrome.setBreadcrumbs([
+      {
+        // using # prevents a reloading of the whole app when clicking the breadcrumb
+        href: '#',
+        text: i18n.translate('home.breadcrumbs.homeTitle', { defaultMessage: 'Home' }),
+        // TODO telemetry for navigating away from getting started via breadcrumbs
+        onClick: () => {},
+      },
+      {
+        text: i18n.translate('home.breadcrumbs.gettingStartedTitle', {
+          defaultMessage: 'Getting Started',
+        }),
+      },
+    ]);
+  }, [chrome]);
+
+  const onSkip = () => {
+    // TODO telemetry for guided onboarding
+    trackUiMetric(METRIC_TYPE.CLICK, 'guided_onboarding__skipped');
+    application.navigateToApp('home');
+  };
+  return (
+    <KibanaPageTemplate template="empty">
+      <EuiPanel className="gettingStarted__panel">
+        <EuiTitle size="l" className="eui-textCenter">
+          <h1>
+            <FormattedMessage
+              id="home.guidedOnboarding.gettingStarted.useCaseSelectionTitle"
+              defaultMessage="What would you like to do first?"
+            />
+          </h1>
+        </EuiTitle>
+        <EuiSpacer size="s" />
+        <EuiText color="subdued" size="s" textAlign="center">
+          <p>
+            <FormattedMessage
+              id="home.guidedOnboarding.gettingStarted.useCaseSelectionSubtitle"
+              defaultMessage="Select an option below to get a quick tour of the most valuable features based on your preferences."
+            />
+          </p>
+        </EuiText>
+        <EuiSpacer size="s" />
+        <EuiSpacer size="xxl" />
+        <EuiFlexGrid columns={3} gutterSize="xl">
+          <EuiFlexItem>
+            <UseCaseCard useCase="search" />
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <UseCaseCard useCase="observability" />
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <UseCaseCard useCase="security" />
+          </EuiFlexItem>
+        </EuiFlexGrid>
+        <EuiSpacer />
+        <EuiHorizontalRule />
+        <EuiSpacer />
+        <div className="eui-textCenter">
+          <EuiLink onClick={onSkip}>
+            <FormattedMessage
+              id="home.guidedOnboarding.gettingStarted.skip.buttonLabel"
+              defaultMessage="I'd like to do something else (Skip)"
```

Comment:
To me, the option to "...do something else {Skip}" suggests that there will be follow-up questions to guide the user through a customized onboarding tour using a Q&A type interaction rather than skipping the tour completely.
Maybe we should be more explicit about what choosing to 'do something else' means: 
```suggestion
              defaultMessage="I want to skip the tour"
```

Reply:
Thanks a lot for your review, @TinaHeiligers! I agree with your comment and would like to get Kelly's @kellyemurphy input on the concrete wording here.